.goreleaser.yml

```diff
@@ -5,117 +5,133 @@ project_name: HORNET
 
 # Environment variables
 env:
-- GO111MODULE=on
+  - GO111MODULE=on
 
 # Builds
 builds:
-# macOS AMD64
-- id: hornet-darwin-amd64
-  binary: hornet
-  env:
-  - CGO_ENABLED=1
-  - CC=o64-clang
-  - CXX=o64-clang++
-  ldflags:
-  - -s -w -X github.com/gohornet/hornet/plugins/cli.AppVersion={{.Version}}
-  flags:
-  - -tags=pow_avx
-  main: main.go
-  goos:
-    - darwin
-  goarch:
-    - amd64
-# Linux AMD64
-- id: hornet-linux-amd64
-  binary: hornet
-  env:
-  - CGO_ENABLED=1
-  ldflags:
-  - -s -w -X github.com/gohornet/hornet/plugins/cli.AppVersion={{.Version}}
-  flags:
-  - -tags=pow_avx
-  main: main.go
-  goos:
-    - linux
-  goarch:
-    - amd64
-# Linux ARM(v7)
-- id: hornet-linux-arm
-  binary: hornet
-  env:
-    - CGO_ENABLED=1
-    - CC=arm-linux-gnueabi-gcc
-  ldflags:
-  - -s -w -X github.com/gohornet/hornet/plugins/cli.AppVersion={{.Version}}
-  flags:
-  - -tags=pow_c
-  main: main.go
-  goos:
-    - linux
-  goarch:
-    - arm
-  goarm:
-    - 7
-# Linux ARM64
-- id: hornet-linux-arm64
-  binary: hornet
-  env:
-    - CGO_ENABLED=1
-    - CC=aarch64-linux-gnu-gcc
-  ldflags:
-  - -s -w -X github.com/gohornet/hornet/plugins/cli.AppVersion={{.Version}}
-  flags:
-  - -tags=pow_c
-  main: main.go
-  goos:
-    - linux
-  goarch:
-    - arm64
-# Windows AMD64
-- id: hornet-windows-amd64
-  binary: hornet
-  env:
-    - CGO_ENABLED=1
-    - CC=x86_64-w64-mingw32-gcc
-    - CXX=x86_64-w64-mingw32-g++
-  ldflags:
-  - -s -w -X github.com/gohornet/hornet/plugins/cli.AppVersion={{.Version}}
-  flags:
-  - -tags=pow_avx
-  main: main.go
-  goos:
-    - windows
-  goarch:
-    - amd64
+  # macOS AMD64
+  - id: hornet-darwin-amd64
+    binary: hornet
+    env:
+      - CGO_ENABLED=1
+      - CC=o64-clang
+      - CXX=o64-clang++
+    ldflags:
+      - -s -w -X github.com/gohornet/hornet/plugins/cli.AppVersion={{.Version}}
+    flags:
+      - -tags=pow_avx
+    main: main.go
+    goos:
+      - darwin
+    goarch:
+      - amd64
+  # Linux AMD64
+  - id: hornet-linux-amd64
+    binary: hornet
+    env:
+      - CGO_ENABLED=1
+    ldflags:
+      - -s -w -X github.com/gohornet/hornet/plugins/cli.AppVersion={{.Version}}
+    flags:
+      - -tags=pow_avx
+    main: main.go
+    goos:
+      - linux
+    goarch:
+      - amd64
+  # Linux ARM(v7)
+  - id: hornet-linux-arm
+    binary: hornet
+    env:
+      - CGO_ENABLED=1
+      - CC=arm-linux-gnueabi-gcc
+    ldflags:
+      - -s -w -X github.com/gohornet/hornet/plugins/cli.AppVersion={{.Version}}
+    flags:
+      - -tags=pow_c
+    main: main.go
+    goos:
+      - linux
+    goarch:
+      - arm
+    goarm:
+      - 7
+  # Linux ARM64
+  - id: hornet-linux-arm64
+    binary: hornet
+    env:
+      - CGO_ENABLED=1
+      - CC=aarch64-linux-gnu-gcc
+    ldflags:
+      - -s -w -X github.com/gohornet/hornet/plugins/cli.AppVersion={{.Version}}
+    flags:
+      - -tags=pow_c
```

Comment:
Does pow_arm_c128 work here?

Reply:
I have submitted a PR to iota.go (iotaledger/iota.go#138)
After it's merge we can switch to `pow_arm_c128`